app/views/list/_list_row_template.slim

```diff
@@ -0,0 +1,10 @@
+tr.request
+  td.request-checkbox
+    input[type="checkbox"]
```

Comment:
Not entirely sure how to properly implement this, could you give me some more info? I get implementing the two inputs and the names, but I don't know enough to understand how it works for the case you outlined